bears/general/FilenameBear.py

```diff
@@ -43,6 +45,10 @@ def run(self, filename, file,
         :param ignore_uppercase_filenames:
             Whether or not to ignore fully uppercase filenames completely,
             e.g. COPYING, LICENSE etc.
+        :param prefix:
+            Check whether the filename uses a certain prefix.
+        :param suffix:
+            Check whether the filename uses a certain suffix.
```

Comment:
You should definitely mention that the file ending is not checked. Otherwise I would have though that I need to include the file-ending^^ (as a filename consists of `name.ending`)